Project.toml

```diff
@@ -35,7 +35,7 @@ EnzymeStaticArraysExt = "StaticArrays"
 BFloat16s = "0.2, 0.3, 0.4, 0.5"
 CEnum = "0.4, 0.5"
 ChainRulesCore = "1"
-EnzymeCore = "0.8.4"
```

Comment:
can the compat be set to either 0.8.4 or 0.8.5 ?

Reply:
do you mean to support both versions? we can put the `~` specifier in the version

Reply:
yup so "0.8.4, 0.8.5"